fleet-state/fleet-state.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+NAMESPACE=$1
+
+if [ -z $NAMESPACE ];
+then
+    echo "Usage: ./fleet-state.sh <namespace>"
+    exit 1
+fi
+
+kubectl get namespace $NAMESPACE 1>/dev/null 2>&1
+if [ $? -ne 0 ];
+then
+    echo "Namespace ${NAMESPACE} does not exist"
+    exit $?
```

Comment:
This will always return 0 since it has the exit code of the echo statement above instead of the output of `kubectl`

Also I think I'm still in the CircleCI-Public/Server notify list :D

Reply:
Me too ❤️ Hey there CircleCI alum!

Reply:
So am I :D Hey, folks!

Reply:
SO much ❤️  to see y'all here.

Reply:
Lmao thank you, nice catch on the wrong exit code
❤️